dev-support/Jenkinsfile

```diff
@@ -60,7 +60,7 @@ pipeline {
     ASF_NIGHTLIES_BASE = "${ASF_NIGHTLIES_BASE_ORI.replaceAll(' ', '%20')}"
     // These are dependent on the branch
     HADOOP3_VERSIONS = "3.3.5,3.3.6,3.4.0"
```

Comment:
should add 3.4.1 to this line too?

Reply:
Yes we do, otherwise the caching won't work.

The problem is that backward compatibility test will run for 16 hours now.

I should probably exclude the default version from the backwards compatibility tests.

Reply:
Added